internal/core/entities/scheduler.go

```diff
@@ -43,12 +44,28 @@ const (
 )
 
 type Scheduler struct {
-	Name            string `validate:"min=1"`
-	Game            string `validate:"min=1"`
-	State           string
+	Name            string `validate:"required"`
+	Game            string `validate:"required"`
+	State           string `validate:"required"`
 	RollbackVersion string
 	Spec            game_room.Spec
 	PortRange       *PortRange
 	CreatedAt       time.Time
-	MaxSurge        string `validate:"min=1"`
+	MaxSurge        string `validate:"required"`
```

Comment:
@lyracampos if i remeber correctly, this maxsurge field can be either an empty string, or an integer number, or a percentage (number with suffix %, e.g 5%), what do you think about including this validation here? you can see how we parse it with more details here: internal/core/services/room_manager/room_manager.go:278

Reply:
@GuilhermBrSp good! I'll create a new custom validation for this field